assets/images/posts/2020-08-06-how-i-schooled-my-seniors/feature-image.svg

```diff
@@ -0,0 +1,565 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
```

Comment:
Unfortunately, the svg is missing the Poppins font used for the tagline. Perhaps, we'll need to use a PNG in this case.